build.gradle

```diff
@@ -40,6 +40,8 @@ ext {
     scalaMajDotMin = "$scalaMajor.$scalaMinor"
 }
 
+task('buildAndTestAll', description: 'Builds all the sub-projects as well as runs there tests.', group: 'release')
```

Comment:
`buildAndCheckAll`? 😛 

I honestly don't know what else to call this task. It doesn't do anything on its own, but running it would run `build` on all sub-projects due to dependencies. I suppose you could call it `publishGate`, but that is more descriptive of what it is than what it does.